src/extension.ts

```diff
@@ -1,10 +1,13 @@
-import { Prisma } from '@prisma/client/extension.js'
+import { Prisma, PrismaClient } from '@prisma/client/extension.js'
 
 import { ConfigureReplicaCallback, ReplicaManager } from './ReplicaManager'
 
-export type ReplicasOptions = {
-  url: string | string[]
-}
+export type ReplicasOptions =
+  | {
+      url: string | string[]
+      replicas?: PrismaClient[]
+    }
+  | { url?: string | string[]; replicas: PrismaClient[] }
```

Comment:
I'm wondering if it's acceptable to also allow accepting both replicaUrls and replicas simultaneously.

Reply:
I think it should not be allowed and specifying both should result in both compile- and runtime error.

Reply:
Thank you for your comment! I've updated it to accept only one of `url` or `replicas`.